_pages/kriskirov.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kriskirov
+pageUrl: kriskirov.github.io
+timestamp: TODAYS-24-2017-04-Mo
+---
```

Comment:
@brianamarie I understand that I have provided the wrong URL for the link, but the URL provided in my Settings tab is of another repository. Is there no way to add a link to a non-repository?